package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-ebates",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
This bump is already merged onto `master`. Why is it still here on the PR diff? 🤔